controller/src/test/java/io/pravega/controller/auth/PasswordEncryptorTool.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.auth;
+
+import org.jasypt.util.password.StrongPasswordEncryptor;
+
+public class PasswordEncryptorTool {
```

Comment:
Are these tools needed? If so, please add some Javadoc as to how they might be used.